### DISCUSSION TITLE
Unify order book filtering with notification alerts

### DISCUSSION THREAD

**Discussion by philipmw** [bisq-network/bisq#6960] [Ideas]
As a new user, I am eligible to trade only up to 0.01 BTC. But there's no way to configure notification alerts for a maximum amount, so I get alerts for offers I am not eliglbe for. (Related to #6090.)

Tangentially, I'd like to filter the order books by only offers I am interested in, such as those at <= 0% premium to the market price. It serves no purpose to show me offers at 7% premium since I have no plans to trade those. It's just noise on my screen.

These two desires can be combined into one logical feature: add filtering to the order book screens and notify me of all new offers that match my filters.

The outcomes of this potential feature:
1. I am notified of all offers that I see on my screen;
2. I am notified only of offers that I am eligible for;
3. I am not notified of offers I am not interested in.

**Comment by ghost** [comment]
I see the benefits of this.  :+1:

Your proposal: _"add filtering to the order book screens and notify me of all new offers that match my filters."_

- Are you proposing migrating the Account/Notifications screen to a more flexible filtering & notifications setup?  
- There are currently 3 filtering options on the BUY/SELL offer list screens: `currency`, `payment method` and `Offers matching my accounts`.  In addition, the option `Ignored peers` in settings filters the offer list.  I'm hoping your filtering would move those existing configurations to a dedicated screen with more options.  
- Some users might want notifications without filtering their display.  Others might want filtering of the display but no mobile notifications.  How would you cater to this?  Would you be willing to propose a mock up drawing of that screen?
- Where in the menus would your filtering configuration be?
- Would the filtering affect the the Buy / Sell books and the Market Offer Book or just the former?


Its not my call whether TPTB will allow new features for Bisq1, but in my opinion this seems worthy.
